x-pack/plugin/stack/src/javaRestTest/java/org/elasticsearch/xpack/stack/EcsDynamicTemplatesIT.java

```diff
@@ -182,6 +182,38 @@ public void testNestedFields() throws IOException {
         verifyEcsMappings(indexName);
     }
 
+    public void testNumericMessage() throws IOException {
```

Comment:
@felixbarny @axw It would be nice if in this (or another) PR we could add the test around the message field. We might make changes in the future and miss the discussion we just had in this PR and the potential effects of it.

Reply:
I don't understand. This test is related to the `message` field. What would you like to see added?

Reply:
🤦‍♂️ Too early. Ignore me.